src/urllib3/exceptions.py

```diff
@@ -382,6 +382,9 @@ class UnrewindableBodyError(HTTPError):
 class NameResolutionError(HTTPError, socket.gaierror):
     """Raised when host name resolution fails."""
 
-    def __init__(self, host: str, reason: socket.gaierror):
-        message = f"Failed to resolve '{host}' ({reason})"
-        HTTPError.__init__(self, message)
+    original_error: Exception
+
+    def __init__(self, host: str, error: Exception) -> None:
```

Comment:
Not sure if we need `original_error` on this exception?

Reply:
I removed it